table/server/master/src/main/java/alluxio/master/table/DbConfig.java

```diff
@@ -55,153 +97,537 @@ public DbConfig(@JsonProperty("bypass") @Nullable BypassEntry bypassEntry) {
    * @return an empty config instance
    */
   public static DbConfig empty() {
-    return new DbConfig(new BypassEntry(Collections.emptySet()));
+    return new DbConfig(null, null);
   }
 
   /**
-   * @return the {@link BypassEntry} from config file
+   * @return the {@link TablesEntry} for bypassed tables from config file
    */
-  public BypassEntry getBypassEntry() {
+  public TablesEntry<TableEntry> getBypassEntry() {
     return mBypassEntry;
   }
 
   /**
-   * @return the {@link UdbBypassSpec} object
+   * @return the {@link TablesEntry} for ignored tables from config file
    */
-  public UdbBypassSpec getUdbBypassSpec() {
-    return mBypassEntry.toUdbBypassSpec();
+  public TablesEntry<NamePatternEntry> getIgnoreEntry() {
+    return mIgnoreEntry;
+  }
+
+  /**
+   * @return the {@link UdbMountSpec} object
+   */
+  public UdbMountSpec getUdbMountSpec() {
+    UdbMountSpec.Builder builder = new UdbMountSpec.Builder();
+    for (TableEntry entry : mBypassEntry.getList().getIncludedEntries()) {
+      if (entry.isPattern()) {
+        builder.bypass().include().addPattern(entry.getPattern());
+        continue;
+      }
+      if (entry.getPartitions().isEmpty()) {
+        builder.bypass().include().addName(entry.getName());
+        continue;
+      }
```

Comment:
`entry` can be:
* a plain name
* a regex
* a table-partition specification

When entry is regex, it does not have a specific name (`getName` or `getTable` returns `null`), so we deal with this  case first. Otherwise, when partition specification is absent, simple add the table name.